From bdc674ef05f8114d41bd6f9248bb173f85a2db9f Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Thu, 23 Feb 2017 20:59:44 +0100
Subject: [PATCH 2/2] Revert "Revert "Add geometry_msgs dependency""

This reverts commit ba2e0c1737bb7a0175da5b565420102620abcbdc.

j-rivero: The patch was still needed to I revert the revert here. Not
nice, I know.
---
 package.xml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/package.xml b/package.xml
index c2a5c26..3b6388a 100644
--- a/package.xml
+++ b/package.xml
@@ -12,10 +12,12 @@
 
   <author email="dstephen@ihmc.us">Doug Stephen</author>
 
-  <build_depend>message_generation</build_depend>
-  <run_depend>message_runtime</run_depend>
   <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>message_generation</build_depend>
   <build_depend>std_msgs</build_depend>
+  <run_depend>geometry_msgs</run_depend>
+  <run_depend>message_runtime</run_depend>
   <run_depend>std_msgs</run_depend>
 
 </package>
-- 
2.7.4

